odoo/tools/pdf.py

```diff
@@ -12,6 +12,7 @@
 from reportlab.lib.units import cm
 from reportlab.lib.utils import ImageReader
 from reportlab.pdfgen import canvas
+from packaging import version
```

Comment:
Aren't we supposed to use our own `parse_version`?
Not sure what the implications are

Reply:
I don't think it will change anything since the versions follow the simple schema "x.y.z", but let's use the odoo `parse_version` anyway. I just pushed a new commit.